Cognite.Simulator.Utils/Configuration.cs

```diff
@@ -43,10 +43,10 @@ public class SimulatorLoggingConfig
     }
 
     /// <summary>
-    /// Represents the configuration for a file library. Used to configure <seealso cref="FileLibrary{T, U}"/>
+    /// Represents the configuration for a model library. Used to configure <seealso cref="ModelLibraryBase{V}"/>
```

Comment:
Should we ignore the Github warning ? 

`XML comment has cref attribute 'ModelLibraryBase{V}' that could not be resolved`